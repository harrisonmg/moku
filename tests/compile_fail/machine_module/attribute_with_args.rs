use moku::*;

#[state_machine]
mod blinky {
    use moku::*;

    #[machine_module()]
    mod state_machine_module {}
}

fn main() {}
