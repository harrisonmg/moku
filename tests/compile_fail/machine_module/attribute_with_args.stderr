error: moku `machine_module` accepts no arguments, try `#[machine_module]`
 --> tests/compile_fail/machine_module/attribute_with_args.rs:7:5
  |
7 |     #[machine_module()]
  |     ^
